File: backend/src/main/java/com/lawencon/elearn/dto/attendance/AttendanceInsertReqDto.java
package com.lawencon.elearn.dto.attendance;

public class AttendanceInsertReqDto {
	private Long classDtlId;
	private Long learningScheduleId;
	private Long quizScheduleId;
	private Long examScheduleId;
	public Long getClassDtlId() {
		return classDtlId;
	}
	public void setClassDtlId(Long classDtlId) {
		this.classDtlId = classDtlId;
	}
	public Long getLearningScheduleId() {
		return learningScheduleId;
	}
	public void setLearningScheduleId(Long learningScheduleId) {
		this.learningScheduleId = learningScheduleId;
	}
	public Long getQuizScheduleId() {
		return quizScheduleId;
	}
	public void setQuizScheduleId(Long quizScheduleId) {
		this.quizScheduleId = quizScheduleId;
	}
	public Long getExamScheduleId() {
		return examScheduleId;
	}
	public void setExamScheduleId(Long examScheduleId) {
		this.examScheduleId = examScheduleId;
	}
}

File: frontend/src/app/dto/attendance/attendance.insert.req.dto.ts
export class AttendanceInsertReqDto {
    private classDtlId! : number
	private learningScheduleId! : number
	private quizScheduleId! : number
	private examScheduleId! : number

    public get _classDtlId(): number {
        return this.classDtlId;
    }

    public set _classDtlId(_classDtlId: number) {
        this.classDtlId = _classDtlId;
    }

    public get _learningScheduleId(): number {
        return this.learningScheduleId;
    }

    public set _learningScheduleId(_learningScheduleId: number) {
        this.learningScheduleId = _learningScheduleId;
    }

    public get _quizScheduleId(): number {
        return this.quizScheduleId;
    }

    public set _quizScheduleId(_quizScheduleId: number) {
        this.quizScheduleId = _quizScheduleId;
    }

    public get _examScheduleId(): number {
        return this.examScheduleId;
    }

    public set _examScheduleId(_examScheduleId: number) {
        this.examScheduleId = _examScheduleId;
    }

}